server/src/schema.js

```diff
@@ -101,6 +101,7 @@ export const Schema = [`
   }
 
   type Event {
+    group: Group!
```

Comment:
put this after `user` maybe? It looks out of place as the first thing - especially because it's before `id`.